### PR TITLE
add doc: How to catch crash on ios use xcode

### PR DESCRIPTION
**Pull request by qfrank**
[link to view the doc](https://github.com/status-im/status-mobile/blob/4f1ea49454b37b339d3b90ba9b529935094308ca/doc/how-to-catch-crash-on-ios.md)

status: ready

### CODE REVIEW

**Review by Samyoul — APPROVED** [review]
Thank you for detailing this @qfrank

**Review by mohsen-ghafouri — APPROVED** [review]
Thanks @qfrank

**Review by flexsurfer — APPROVED** [review]

**Review by BalogunofAfrica — APPROVED** [review]

**Review by ilmotta — APPROVED** [review]
I can't test the instructions, but thank you anyway @qfrank.

As a suggestion, I did a pass on gpt only to fix the grammar and broke the first paragraph into a few different ones.

```markdown
Sometimes, we can't reproduce a crash with a local dev build, probably because
your built version differs from the one the QA team used. The QA team mostly
used the PR build version. In this guide, we will try to make a build as close
as possible to the PR build for iOS.

You may notice various configuration files starting with .env (such as .env,
.env.e2e, .env.jenkins) in the project. The usage of these files and environment
variables injected at build time on CI will not be covered in this guide.

This guide covers how to configure Xcode to capture a crash stack trace with a
physical iOS device. This allows debugging crashes by inspecting the stack
trace.

We will use [issue
#17255](https://github.com/status-im/status-mobile/issues/17255) as an example
to reproduce and analyze a crash.

```

File: doc/how-to-catch-crash-on-ios.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+We will use [issue #17255](https://github.com/status-im/status-mobile/issues/17255) as an example to reproduce and analyze a crash.
+
+## Prerequisites
+- A physical ios device (simulator is not considered in this doc)
+- Xcode installed
+               
+## Steps
+1. Prepare Status Mobile
+   1. Open Terminal, cd to project root
+   2. Run make run-clojure in one tab
```
Comment (siddarthkay) [inline doc/how-to-catch-crash-on-ios.md]:
```suggestion
   2. Run `make run-clojure` in one tab
```

File: doc/how-to-catch-crash-on-ios.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+   3. Run make run-metro in a second tab
```
Comment (siddarthkay) [inline doc/how-to-catch-crash-on-ios.md]:
```suggestion
   3. Run `make run-metro` in a second tab
```

File: doc/how-to-catch-crash-on-ios.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+   4. Run make pod-install in a third tab
```
Comment (siddarthkay) [inline doc/how-to-catch-crash-on-ios.md]:
```suggestion
   4. Run `make pod-install` in a third tab
```

**Review by siddarthkay — APPROVED** [review]
Other than a few tiny suggestions this document looks good!
Thanks for sharing this knowledge @qfrank !

**Review by jo-mut — APPROVED** [review]

**Comment by jo-mut** [comment]
its a good one

**Review by smohamedjavid — APPROVED** [review]
❤️

**Review by ajayesivan — APPROVED** [review]
@qfrank Nice work 🎉 !

**Comment by qfrank** [comment]
Sorry team, i've made a bad example that merged PR without e2e test although no code changes in this PR! I should also **notice QA team in this case**.  🙂

**Pull request merged**